manifests/1.4.0/opensearch-1.4.0-test.yml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
Do we want the test for `OpenSearch` now since we don't really test anything here?